src/main/java/com/puppycrawl/tools/checkstyle/checks/metrics/AbstractClassCouplingCheck.java

```diff
@@ -72,6 +72,8 @@
             "List", "ArrayList", "Deque", "Queue", "LinkedList",
             "Set", "HashSet", "SortedSet", "TreeSet",
             "Map", "HashMap", "SortedMap", "TreeMap",
+            // Annotations
+            "Override",
```

Comment:
You added `Override` annotation, but what about others? Should we add them?
https://docs.oracle.com/javase/8/docs/api/java/lang/package-summary.html
> Deprecated, SafeVarargs, SuppressWarnings